### ISSUE TITLE
Accessibility: Ensure all page content is contained by landmarks

### ISSUE THREAD

**Issue by damienrobson-sage** [gpbl/react-day-picker#2627]
The table with class `.rdp-month_grid` generates the following `Moderate` WCAG violation when ran via AXE

## To reproduce

Using the [RDP Playground](https://daypicker.dev/playground), run an AXE scan.

## Actual Behavior

A violation of `moderate` level is thrown

## Expected Behavior

No violations are thrown.

## Screenshots

<img width="1609" alt="Screenshot 2024-12-16 at 15 11 05" src="https://github.com/user-attachments/assets/8c75f753-85c9-402f-a662-66e37f2e097b" />

<img width="1330" alt="Screenshot 2024-12-16 at 15 13 07" src="https://github.com/user-attachments/assets/df038a50-74db-4de7-88ad-ef57b962d5a3" />

**Comment by gpbl** [comment]
This is a website issue, not related to DayPicker. Thanks for the report!
